packager/VisibilityChecker.cc

```diff
@@ -297,35 +297,18 @@ class VisibilityCheckerPass final {
 public:
     const core::packages::PackageInfo &package;
     const bool insideTestFile;
+    UnorderedSet<core::SymbolRef> defs;
```

Comment:
Can we use a more descriptive name and/or add a comment with what the meaning is?